x-pack/test/functional/page_objects/lens_page.ts

```diff
@@ -121,6 +122,64 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
       }
     },
 
+    /**
+     * Drags field to workspace
+     *
+     * @param field  - the desired field for the dimension
+     * */
+    async dragFieldToWorkspace(field: string) {
+      await browser.html5DragAndDrop(
+        testSubjects.getCssSelector(`lnsFieldListPanelField-${field}`),
+        testSubjects.getCssSelector('lnsWorkspace')
+      );
+      await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
I'm bot sure we can rely on global spinner here. Is there any reliable way to wait viz loading is completed?

Reply:
Do you have a specific reason for thinking that @dmlemeshko? It has always been accurate before. But @mbondyra I'm not actually sure we need this just to check that the configuration is set correctly, the text is updated before the loading finishes. I would guess that it succeeds without this line.